Dockerfile.release

```diff
@@ -7,17 +7,23 @@ RUN \
      apk add --no-cache --virtual .build-deps curl && \
      echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
      curl https://dl.minio.io/server/minio/release/linux-amd64/minio > /usr/bin/minio && \
-     chmod +x /usr/bin/minio && apk del .build-deps
+     chmod +x /usr/bin/minio
 
 EXPOSE 9000
 
 COPY buildscripts/docker-entrypoint.sh /usr/bin/
 
-RUN chmod +x /usr/bin/docker-entrypoint.sh
+COPY buildscripts/healthcheck.sh /usr/bin/
```

Comment:
@nitisht, we can combine 
COPY buildscripts/docker-entrypoint.sh /usr/bin/
COPY buildscripts/healthcheck.sh /usr/bin/

as COPY buildscripts/docker-entrypoint.sh  buildscripts/healthcheck.sh  /usr/bin  to reduce layers. 

Also, the RUN command for docker-entrypoint.sh can be  folded into the first RUN command if the COPY and EXPOSE commands are moved up further in the script.